sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -159,6 +160,14 @@ abstract class QueryTest extends PlanTest {
     checkAnswer(df, expectedAnswer.collect())
   }
 
+  protected def checkAnswer(
```

Comment:
If it's only used in one test suite, let's move it there. We can move it to `QueryTest` if we see more and more suites using it.

Reply:
I created another suite called `UpdateFieldsPerformanceSuite` as part of this PR where I reuse this function. 
If you want, I could put the contents of `UpdateFieldsPerformanceSuite` in `ColumnExpressionSuite` and then I would be able to keep this function (`checkAnswer`) in just `ColumnExpressionSuite`. Just let me know.